### DISCUSSION TITLE
Leave a comment for chance to win sticker swag

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#602] [General]
Some ideas

- Your goals for 2021
- Share your open source project
- Add value
- ...

5 stickers give away today for a trial 🤓 

---

Winners 🎉 

- https://github.com/Vyvy-vi
- https://github.com/seema1711
- https://github.com/dumbcoder7
- https://github.com/dheerajdlalwani
- https://github.com/saviomartin

**Comment by praveenscience** [comment]
Goals

- Resolution: 3840 x 2160!
- [praveenscience/Internship-LMS-FrontEnd](https://github.com/praveenscience/Internship-LMS-FrontEnd)
- We are in the mission of solving a crisis for a lot of small and medium enterprises, who are trying to recruit interns for their projects. This platform aims to onboard new interns, who have applied to the companies and help them understand the process and make the transition smooth.

**Reply by eddiejaoude** [reply]
Great goals, especially "help interns", keep us posted on how it goes 👍 .

**Comment by SuNiL-Chau** [comment]
Goals They are big but not impossible :P

1. Learn and start posts consistency on (YouTube, Linkedin, dev) on Web Technologies & UI/UX
2. Learn Angular / React
3. Learn new technologies like node.js, socket.io, flutter, express.js, docker
4. Apply for GDE program(Google Developer Expert) for web technology & start open source contribution.
5. Read a book daily (at least 20pages per day).
6. Create your portfolio.

**Reply by eddiejaoude** [reply]
Great goals, especially "posts on YouTube ...", keep us posted on how it goes 👍 .

**Comment by dheerajdlalwani** [comment]
My goals for 2021:
 - Transform my body for becoming a healthier person mentally & physically.
 - Being more inclusive to all communities and people.
 - More and more opensource.
 - My portfolio.
 - Learning Flutter and creating a great open source app.
 - Make videos and write a few articles on collaborations in team work.

**Reply by eddiejaoude** [reply]
Great goals, especially "more open source" and "videos / articles", keep us posted on how it goes 👍 .

**Comment by Harshal0902** [comment]
Goals

- Resolution: 500 or more contribution and grow more by learning
- https://github.com/Harshal0902/DSCWOW_Rudra
- Learn new tech stacks
- Expand my network
- Improve my portfolio
- Teach other and learn from them
- Help more and more people

**Reply by eddiejaoude** [reply]
Great goals, especially "teach others and learn from them", keep us posted on how it goes 👍 .

**Comment by Vyvy-vi** [comment]
> Some ideas
Your goals for 2021
Share your open source project
Add value
...
5 stickers give away today for a trial 🤓

A few more ideas, that could add value:

- More ideas on how to do giveaways. (ways to make this challenging)

- List of opensource flagpoints of last year

- Link to other communities and places to propagate OSS and skills

**Reply by Vyvy-vi** [reply]
#### Goals:

- js and front-end libs(fCC curriculum)
- proficiency in C
- more opensource and collaboration
- school stuff
- try to get over my social anxiety

#### OSS project:
- Mine: https://github.com/Vyvy-vi/TearDrops
- A few small projects I like:
https://github.com/gurkult/py-gurklang
https://gurkult.github.io/py-gurklang/
https://github.com/python-discord/snekbox
https://github.com/nhcarrigan/Becca-Lyria
https://github.com/dictu-lang/Dictu
https://github.com/EddieJaoudeCommunity/EddieBot

#### Another OSS community:
- [Gurkult](https://gurkult.com/)
It's small and new(but quite inclusive and friendly)

### Ways to host these giveaways:
1. Some community challenge(live edit the community site with a feature eddie says on a stream- surprise)
2. Community vote(idk might be cool to checkout, which user was found to be most helpful in the community for the entire year)
3. Community art/design competishion
can't think of more ways(comment down here)🤔

**Reply by eddiejaoude** [reply]
Great goals, keep us posted on how it goes 👍 . 

thank you for the suggestions too 👍  - community vote is interesting 🤔

**Comment by RKC3000** [comment]
Goals:-
Good Contribute to Open Source.

No such type of contribution that's why I am here to learn

I am a Frontend web developer and currently learning Data Science

**Reply by eddiejaoude** [reply]
Great goals, especially "contribute to open source", keep us posted on how it goes 👍 .

**Comment by vamsi963601** [comment]
Can we get some Resolutions in here 👍🏼 
---- First of all do not give the stickers to me chose someone else 😎 
---- I need to get more contributions this year like 700 or so 🤔 
---- I need to start working Eddie's Swag already 💛 

And Happy New year Everyone and a **green square is all that matters**

**Reply by eddiejaoude** [reply]
Great goals, especially "contributions", keep us posted on how it goes 👍 .

**Comment by priyadarshu** [comment]
Goals 2021
1. Read more books
2.  Learn Flutter and React js
3.  Contribute to atleast 2 open source projects
4.  Write inspirational and technical blog of my learnings
5.  Expand my connections
6.  Be happy and healthy 😄
7.  Get started with my Startup

**Reply by eddiejaoude** [reply]
Great goals, especially "contribute to open source projects", keep us posted on how it goes 👍 .

**Comment by adityaraute** [comment]
# My goals for 2021:

- Get a job
- Improve on my tech stack
- Engage in more Open Source Projects

# Value:

- Come here for learning, not stickers. 
- ~~I don't have anything really to add. Only to subtract 😂~~


### Adding headings only cuz I think Eddie wanted it 


~~Sorry Eddie~~

**Reply by eddiejaoude** [reply]
Great goals, especially "engage in open source projects", keep us posted on how it goes 👍 .

**Reply by adityaraute** [reply]
Not adding value technologically as of yet. I'm the guy that forces everyone to improve documentation. I wonder how many maintainers like that though 😹

**Reply by Vyvy-vi** [reply]
tbh, I think documentation work is much appreciated since many maintainers tend to not notice the gaps in the docs, until a fresh pair of eyes notice some discrepancy. 
Moreover, in OSS the documentation is as important(if not more) as the code, considering how docs help maintainers to transition from 'get this X new feature added' to 'add X in such a way that adding new features is easier'

**Comment by codingknite** [comment]
My Goals for 2021
 - Get A Job in 2021
 - Learn Typescript, Node and Express
 - Write one blog a week
 - Start actively contributing to open source

Open Source Project
- https://github.com/mojpm/frontend-development 
- It's a collection of resources for frontend development to help those learning frontend development
- Got 1.2k stars yaay

Value
- I'd love to Mentor 5 people starting out

**Reply by eddiejaoude** [reply]
Great goals, especially "mentor people", keep us posted on how it goes 👍 .

**Comment by himanshu007-creator** [comment]
goal:- to learn new things [android app development] and introduce my college students to opensource
Project:-<a href="https://github.com/himanshu007-creator/DocDial">Docdial</a> a medical consultancy website .

**Reply by eddiejaoude** [reply]
Great goals, especially "introduce college students to open source", keep us posted on how it goes 👍 .

**Comment by nasrob** [comment]
My goals for this year :

- Improving my algorithms, data structures skills, and design patterns.
- More open-source contributions.
- Relaunch my blog

**Reply by eddiejaoude** [reply]
Great goals, especially "open source contributions", keep us posted on how it goes 👍 .

**Comment by bucharitesh** [comment]
Goals 2021

Read more books
Learn Flutter
Start writing Blogs
Expand my connections more
Contribute to at least 2 open source projects
*TRAVELLLLL*

**Reply by eddiejaoude** [reply]
Great goals, especially "open source project" and  "travel", keep us posted on how it goes 👍 .

**Comment by codedoc7** [comment]
Goals 2021
1)Improving Data Science & Web Development skills
2)Practice Practice & Practice
3)Share Knowledge

**Reply by eddiejaoude** [reply]
Great goals, especially "share knowledge", keep us posted on how it goes 👍 .

**Comment by iseemasaharan** [comment]
Goals:
- Help 10k people in tech. For that, I made a community. 
- Make 100 YouTube videos.
- Write more.
- Mentor more people.

**Reply by dheerajdlalwani** [reply]
I would love to join you in helping the community

**Reply by iseemasaharan** [reply]
It's open. I will send you the link.

**Reply by priyadarshu** [reply]
I would like to be a part of your community. Getting into the tech industry (career transition) as a fresher, seems more of a nightmare especially during this pandemic. Thanks for this, looking forward to join you😄

**Reply by eddiejaoude** [reply]
Great goals, especially "help 10k people in tech", keep us posted on how it goes 👍 .

and great collaboration by awesome people 💪

**Reply by iseemasaharan** [reply]
Thank you, Eddie.

**Comment by Akankshabhasin** [comment]
My goals for 2021

- Learn React
- Read a book and write something daily
- Meditate and dance daily
- Build an opensource project
- Get a good job
- Help my juniors in getting into tech.

**Reply by eddiejaoude** [reply]
Great goals, especially "build anon source project", keep us posted on how it goes 👍 .

**Comment by saviomartin** [comment]
Hello there,

My goals for 2021 not only include coding and web development but also I have tried to include my fitness as my goal. 

1. I am gonna start learning android app development and I hope I could do it well. 
1. I will spend some time creating a very good portfolio and a resume. I believe they are much necessary to have a good job.
1. I will spend my time for others, I mean I would share my resources, my expertise, tips tricks, so I could help others too
1. I will make a very good time table for me to follow and will really work very hard to do according to it. 
1. I will try to contribute to more open source project and work to improve my GitHub contribution panel
1. Run 60 rounds every day to improve my productivity
1. Take a break of 5 mins to help my eyes
1. Read at least a book every month
1. Create a very good Github profile and LinkedIn
1. Try to earn $30 every month through freelancing

**Reply by aeyshubh** [reply]
> Some ideas
> 
>     * Your goals for 2021
> 
>     * Share your open source project
> 
>     * Add value
> 
>     * ...
> 
> 
> 5 stickers give away today for a trial nerd_face

Contribute in Stockinator Repo and learn Java script and give a tech talk at Dev Fest .

**Reply by eddiejaoude** [reply]
@saviomartin Great goals, especially "contribute more to open source projects", keep us posted on how it goes 👍 .

**Comment by iason9** [comment]
my name in youtube is Jason changed it in github with my real name..
My goals : 🥅
- 💻 Contribute more to open source and get at least one green square per day at best 3 to 5 contributions per day.
- 💵 Freelance on Upwork
- 💰 make passive income with code and SAAS
- 📖 read more books on coding 6 until the summer
- 📚read 10 books about other field like philosophy and spitituality
- 🚴‍♂️ get more in shape

**Reply by eddiejaoude** [reply]
Great goals, especially "freelance on upwork", keep us posted on how it goes 👍 .

**Reply by iason9** [reply]
You are a great guy responding to every comment!! @eddiejaoude it does make a difference thank you for doing that ❤️

**Reply by eddiejaoude** [reply]
Thank you for your awesome reply @iason9 ❤️ . I try to reply as must as possible, awesome feedback like this gives me the energy and motivation to do it more 👍 

*(although I am 2 weeks behind with notifications, so sorry everyone else, I am trying to catch up 🤦‍♂️ )*

**Comment by KeshavTiwari23** [comment]
Want to contribute more in open source community.
Want to learn bug bounty 
As I am cse student I want to make my coding skills much better and get placed in good mnc.

**Reply by eddiejaoude** [reply]
Great goals, especially "what to contribute more to open source community", keep us posted on how it goes 👍 .

**Comment by souravsingpardeshi** [comment]
my goals
learn networking
learn blockchain
build some projects for resume

**Reply by eddiejaoude** [reply]
Great goals, especially "build projects for resume", keep us posted on how it goes 👍 .

**Comment by neelraval13** [comment]
- I want to learn and also be able to teach people
- https://github.com/neelraval13/News
Don't have anything else though. 

I'm happy today, because I can start Discussions on my repos as well.

**Reply by eddiejaoude** [reply]
Great goals, especially "I want to learn and also to be able to teach people", keep us posted on how it goes 👍 .

**Comment by ghost** [comment]
Goals:
Learn React. Get some good projects
Learn Django
Expand connections
Give time to family and friends

**Reply by eddiejaoude** [reply]
Great goals, especially "give time to family and friends", keep us posted on how it goes 👍 .

**Comment by aeyshubh** [comment]
> Some ideas
> 
>     * Your goals for 2021
> 
>     * Share your open source project
> 
>     * Add value
> 
>     * ...
> 
> 
> 5 stickers give away today for a trial nerd_face



> Some ideas
> 
>     * Your goals for 2021
> 
>     * Share your open source project
> 
>     * Add value
> 
>     * ...
> 
> 
> 5 stickers give away today for a trial nerd_face
The current project i contribute to 
https://github.com/ShubhamPatel33/NeoAlgo. I want to give a Dev talk at Dev Fest and teach others by google and learn React this year .
Next OS project : Stockinator.

**Reply by eddiejaoude** [reply]
Thank you for posting 👍 . Here are some markdown tips in my video, any questions let us know 🤓  https://www.youtube.com/watch?v=-lBcKw3m42U&t=22s

**Comment by DenverCoder1** [comment]
My 2021 Goals

- Finish CompSci degree
- 2k subs on YouTube
- Become active on Twitter
- Consistently update blog
- Finish side projects I started

A couple of my open source projects:

- [LaTeX Gboard Dictionary](https://github.com/DenverCoder1/LaTeX-Gboard-Dictionary)
- [GitHub Readme Streak Stats](https://github.com/DenverCoder1/github-readme-streak-stats)

If anyone is interested in learning JavaScript, I post tips and quizzes on [Twitter](https://twitter.com/DenverCoder1/)  and tutorials on YouTube (["Dev Pro Tips"](https://www.youtube.com/c/DevProTips)).

**Reply by eddiejaoude** [reply]
Great goals, especially "finish side projects I started", keep us posted on how it goes 👍 . I need to do the same, I start so many projects 😂

**Comment by Utkarsh299-tech** [comment]
Goals'2021:

1. Contribute to open source projects as much as I can
2. Start my Youtube channel and teach whatever I'm learning in my college
2. Participate in GSoC'21
3. Build projects based on my learning
4. Make my family proud by doing some great stuff this year
5. Read my favorite books
6. Take guitar lessons

**Reply by eddiejaoude** [reply]
Great goals, especially "start my YouTube channel", keep us posted on how it goes 👍 .

**Reply by Utkarsh299-tech** [reply]
Thank you so much for your reply @eddiejaoude! You're really a great person. I'm very glad that I got a chance to meet you through one of your live streams on YouTube. This is one of the best things that happened to me this year. And I'm enjoying that. I'll surely post regarding my YouTube channel. Thanks once again! 😀👍

**Comment by DebugAgrawal** [comment]
### Few of my Goals to conquer in 2021 🏆


- [ ] 1. To build a global network to help my peers and juniors in every possible aspect.


- [x] 2. To become an MLH fellow. <img src=https://media.tenor.com/images/13b9b8e1fb4e9754dd2becc895edf056/tenor.gif height=3% width=3%>

- [ ] 3. To become an Github campus Expert and bring Github community to my campus.
- [ ] 4. Build a strong network on linkedIn.
- [ ] 5. Crack GSoC this year and become a repo maintainer

**Reply by eddiejaoude** [reply]
Great goals, especially "repo maintainer", keep us posted on how it goes 👍 .

**Reply by DebugAgrawal** [reply]
> one done 😊🥺 (19-01-21)

**Comment by RasPiPkr** [comment]
Goals 2021:

[ ] - Get a job
[ ] - Contribute to open source
[ ] - Make more open source
[ ] - Add to this list......

**Reply by eddiejaoude** [reply]
Great goals, especially "contribute to open source" and "get a job", keep us posted on how it goes 👍 .

**Comment by Nitesh-thapliyal** [comment]
> 2021 Goals
- I will be doing valuable Contributions, will learn more about technology, and will be making more projects.
-  And I will be focusing on learning by doing rather than reading stuff.
- And I will be making my friend learn about open source and make them participate 
- In 2020 I won an open-source event contributing 4 projects related to Hadoop and AWS.
-  In 2021 I will be learning Jenkins and Kubernetes
- And project that I am working on in I voice assistant that can do all tech stuff like configuring webserver and can do aws stuff just by speaking to it.

**Reply by eddiejaoude** [reply]
Great goals, especially "Kubernetes", keep us posted on how it goes 👍 .

**Comment by Somsubhra1** [comment]
Goals for 2021:

- Learn Django and Flask
- Dive deep into ML
- Do more internships

Open source project: https://github.com/Somsubhra1/Video-Conference

TODO: whatever I missed in 2020 due to Covid pandemic. Like travelling, enjoying with friends, etc.

**Reply by eddiejaoude** [reply]
Great goals, especially "travelling, enjoying with friends", keep us posted on how it goes 👍 .

**Comment by muddhit** [comment]
Goals:

- Master MERN stack,learn Gatsby and NextJs
- https://github.com/Mukulbaid63/MausamSamachar
- I am already mentoring hundreds of people out there for paving there way out in the programming world and soon I am going to launch a bigger community which will be a revolution in OpenSource world

**Reply by eddiejaoude** [reply]
Great goals 👍 . But what about mentor other people other than guys? Or is this a typo?

**Reply by muddhit** [reply]
yeah it was a typo🤭

**Reply by eddiejaoude** [reply]
Thanks for updating 🤓

**Comment by a-k-r-a-k-r** [comment]
Goal
Secure CCNA certification

**Comment by elenakout** [comment]
Goals
- Get my CS degree
- Complete my freelancing project
- Learn more
- Spend more time with my family

**Reply by eddiejaoude** [reply]
Great goals, especially "spend ore time with family", keep us posted on how it goes 👍 . I need to do this one more too 🤓

**Comment by samir-0711** [comment]
Goal:
- To be expert at DSA
- Top contributor of open source projects
- https://github.com/samir-0711/SSH-Secure-Shell-Server
-  Give back to community as much as I can

**Reply by eddiejaoude** [reply]
Great goals, especially "top contributor of open source projects", keep us posted on how it goes 👍 .

**Comment by ayescythe** [comment]
Goal

- To learn cybersecurity and improve in competitive coding
- contribute more to open source projects.
- create automated tools using python.

**Reply by eddiejaoude** [reply]
Great goals, especially "contribute to open souce", keep us posted on how it goes 👍 .

**Comment by droidboot** [comment]
My Goals for 2021 -
• Learn Flutter, master Kotlin and Java.
• Get a Job.
• Start contributing to Open source projects.
• Read books.
• Release awesome Android apps and my own social platform which is in development :D
• Be more active on Dev and Github :p

**Reply by eddiejaoude** [reply]
Great goals, especially "be more active on GitHub", keep us posted on how it goes 👍 . This swag competition already finished. There will be more soon 🤓

**Comment by DanBrown47** [comment]
My Goals for 2021 - 

- Build a career not a job
- Earn money by working 
- Volunteer to secure cyberspace not for reward
- Above all make everyone happy 
- Be a human

**Reply by eddiejaoude** [reply]
Great goals, especially "make everyone happy", keep us posted on how it goes 👍 . This swag competition already finished. There will be more soon 🤓

**Comment by shekabhi** [comment]
My Goal for 2021 : 

1 : Become Master on Codeforces
2 : placed in Product Based Company
3 : Contribution on Open Source Projects
5 : Compose A Song 
🔥

**Reply by adityaraute** [reply]
You missed 4

**Reply by shekabhi** [reply]
4th one is secret 😄

**Reply by eddiejaoude** [reply]
lol 😂 

3 is the winner though 💪

**Comment by Keshav04042001** [comment]
Goals:-
Get involved in open source as much as I can.
contributing in 15+ verified organization,1k+ commits , 100+ merged PR's and a lot stars.
Here are some projects in which I contributed and mine private once
  https://github.com/Keshav04042001/awesome-computer-science-opportunities
  https://github.com/Keshav04042001/Finman
  https://github.com/Keshav04042001/My-First-Website
Crack Gsoc'21 and MLH spring 2021
Will go for making YT videos to help community.(If i crack gsoc'21)
Do more CP.

**Reply by eddiejaoude** [reply]
YT videos FTW 👍

**Comment by kitarp29** [comment]
Goals for 2021:
- Get better at my Problem-Solving Skills(C++) and Interview Prep
- Finally, I contribute to some worthy Project in Open-Source
- Get me an Internship
- Participate in an OFFLINE HACKATHON for once,(online one sucks)
- Make a product that for real-life usage

**Reply by eddiejaoude** [reply]
Why do online ones suck?

**Reply by kitarp29** [reply]
The way my seniors describe n OFFLINE HACKATHON, it's super AWESOME!!!
People staying up all night together, fun activities, networking in real life.
Getting away from college for a weekend with Friends!!
My expectations are real high

**Reply by eddiejaoude** [reply]
It is different, both have pros/cons - a big advantage of online hackathon, is the collaboration is unlimited 🤓 . I do miss offline ones too though

**Reply by kitarp29** [reply]
Hope that I get into a hackathon where u are the guest speaker or smthng😊😄😁
So that I can meet you in real life.

**Reply by eddiejaoude** [reply]
Sure, one day 👍 🤞

**Comment by Nitesh-thapliyal** [comment]
Offline Hackathons are great but think of it, in Offline hackathon you are with your friends and a couple of more people whom you know already but Online hackathon gives the opportunity to collaborate with folks from all over the world, that's so cool. I personally like meeting new people, I like to know about their culture and what sort of tech they are interested in, and collaborating with so many folks from all over the world definitely gives us a reach and boost. Though this is my first official Hackathon, in these 6 days I got to learn so much and meet new people from all over the world, formed a team with them, collaborate with them, it is so fun😄

**Reply by eddiejaoude** [reply]
Awesome! Great to hear 👍  Collaboration FTW

**Comment by Kitecatcher** [comment]
goals make an ar style program with a bunch of cool people learn a ton. I would like to eventually build an open-sourced project and share it with the world.  Wanna help?

**Reply by eddiejaoude** [reply]
Go for it 👍 Build it first then try to get people involved 🤓
